### PR TITLE
BF: Fix bug in example of reconst_csd.py

### PR DESCRIPTION
**Pull request by RicciWoo**
This is to fix the error of example reconst_csd.py.
1. transform image data from 1D to 4D before calling actor.odf_slicer();
2. remove unexpected argument ‘scale' when calling actor.peak_slicer().

### CODE REVIEW

File: doc/examples/reconst_csd.py [hunk 1]
```diff
@@ -225,7 +227,7 @@
                              parallel=True)
 
 window.clear(ren)
-fodf_peaks = actor.peak_slicer(csd_peaks.peak_dirs, csd_peaks.peak_values, scale=1.3)
+fodf_peaks = actor.peak_slicer(csd_peaks.peak_dirs, csd_peaks.peak_values)
```
Comment (skoudoro) [inline doc/examples/reconst_csd.py]: Why are you changing the scale here?

File: doc/examples/valid_examples.txt [hunk 2]
```diff
@@ -1,62 +1,62 @@
- quick_start.py
- tracking_quick_start.py
- brain_extraction_dwi.py
- reconst_csa_parallel.py
- reconst_csa.py
- reconst_csd_parallel.py
+# quick_start.py
```
[the remark this answers is not in the document]
Comment (RicciWoo) [reply doc/examples/valid_examples.txt]: file removed.

**Comment by RicciWoo** [comment]
Do I need to remove the change in dti.py, because it's already in PR #1468

File: doc/examples/reconst_csd.py (hunk 1 above)
Comment (RicciWoo) [reply doc/examples/reconst_csd.py]: The same reason apply to PR #1479 in example sfm_reconst.py.

**Pull request closed without merging**